packages/qulacs/quri_parts/qulacs/__init__.py

```diff
@@ -18,3 +19,11 @@
 QulacsParametricStateT: TypeAlias = Union[
     ParametricCircuitQuantumState, ParametricQuantumStateVector
 ]
+
+Numerics = TypeVar("Numerics", int, float, complex)
+
+
+def cast_to_list(int_sequence: Union[Sequence[Numerics], ArrayLike]) -> list[Numerics]:
+    """Cast a sequence of numerics (int, float, complex) or an array to a list
+    of the same type."""
```

Comment:
Could you describe the purpose of this function (workaround for too strict type annotation of Qulacs)?

Reply:
The current docstring is:
"""Cast a sequence of numerics (int, float, complex) or an array to a list
    of the same type.

    This is a workaround for too strict type annotation of Qulacs
"""